client-v2/src/app/components/molecules/entity-description/entity-description.component.ts

```diff
@@ -1,53 +1,73 @@
 import { ChangeDetectionStrategy, Component, Input, Output, ViewChild } from '@angular/core'
-import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy'
-import { BehaviorSubject, filter, first, map, merge, Subject, switchMap, tap } from 'rxjs'
-import { FocusableDirective } from 'src/app/directives/focusable.directive'
-import { createEventEmitter } from 'src/app/utils/observable.helpers'
+import { UntilDestroy } from '@ngneat/until-destroy'
+import {
+    Observable,
+    Subject,
+    delay,
+    distinctUntilKeyChanged,
+    filter,
+    map,
+    mergeMap,
+    shareReplay,
+    switchMap,
+} from 'rxjs'
+import {
+    defaultDesktopEditorLayout,
+    getDefaultEditorFeatures,
+    getDefaultEditorLayout,
+    provideEditorFeatures,
+} from 'src/app/rich-text-editor/features'
+import { TipTapEditorComponent } from 'src/app/rich-text-editor/tip-tap-editor/tip-tap-editor.component'
+import { DeviceService } from 'src/app/services/device.service'
+
+export interface DescriptionContext {
+    id: string
+    description$: Observable<string>
+}
 
 @UntilDestroy()
 @Component({
     selector: 'app-entity-description',
     templateUrl: './entity-description.component.html',
     styleUrls: ['./entity-description.component.css'],
     changeDetection: ChangeDetectionStrategy.OnPush,
+    viewProviders: [provideEditorFeatures(getDefaultEditorFeatures())],
 })
 export class EntityDescriptionComponent {
-    description$ = new BehaviorSubject<string | null>(null)
-    @Input() set description(description: string | null) {
-        this.description$.next(description)
-    }
+    constructor(private deviceService: DeviceService) {}
 
-    @ViewChild(FocusableDirective) focusable!: FocusableDirective // needed for outside access
+    @ViewChild(TipTapEditorComponent) ttEditor!: TipTapEditorComponent
 
-    descriptionChanges$ = new BehaviorSubject<string | null>(null)
-    blurEvents$ = new Subject<FocusEvent>()
-    @Output() blur = createEventEmitter(this.blurEvents$.pipe(tap(this.deselectEditor), untilDestroyed(this)))
+    toolbarLayout = defaultDesktopEditorLayout
+    toolbarLayout$ = this.deviceService.isTouchPrimary$.pipe(map(getDefaultEditorLayout))
 
-    descriptionDomState$ = merge(
-        this.descriptionChanges$,
-        this.description$.pipe(
-            tap(() => {
-                if (this.descriptionChanges$.value !== null) this.descriptionChanges$.next(null)
-            })
-        )
+    private context$ = new Subject<DescriptionContext>()
+    @Input() set context(context: DescriptionContext | null) {
+        if (context) this.context$.next(context)
+    }
+
+    private editorBound$ = this.context$.pipe(
+        distinctUntilKeyChanged('id'),
+        map(({ id, description$ }) => this.ttEditor.bindEditor(description$, id)),
```

Comment:
Lets not use the `bindEditor()` method, but rather make it private in the editor and use the template binding syntax. (Makes life easier if the editor isn't initialised yet)